docs/crds/l7rule.md

```diff
@@ -0,0 +1,112 @@
+### L7Rule 
+
+L7Rule CRD can be used to modify the default properties of the L7 VS created using the Hostrule CRD. [`AKOSettings.enableEVH`](../akoconfig.md#akoconfig-custom-resource) needs to be set to `true` to enable this feature. 
```

Comment:
There is empty union between Hostrule and L7 rule properties.  We can say that it is used to override or modify VS properties which are not part of Hostrule.

Reply:
L7Rule is applicable when AKO is running in EVH mode.

Reply:
changed